### PR TITLE
Breaking: TreePicker Svg Icons replacing imgs

### PR DESCRIPTION
**Pull request by camjc**
- Uses latest Alexandria, so exports SvgSymbol,
  SvgSymbolCircle.

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

**Comment by vinteo** [comment]
when we put this into direct-web will need to make sure the styles for the product mapper empty state still holds

File: scripts/commit-msg [hunk 1]
```diff
@@ -46,7 +47,7 @@ function checkGitCommit() {
   }
 
   // Only check non-release messages
```
Comment (omgaz) [inline scripts/commit-msg]: only check non-release messages and ignore greenkeeper style chores with super-long package names

**Comment by omgaz** [comment]
Merging :ocean:

**Pull request merged**